integration/coverage_test.go

```diff
@@ -59,23 +58,24 @@ var _ = Describe("Coverage Specs", func() {
 		Eventually(session).Should(gexec.Exit(0))
 
 		// Check that the correct file was created
-		_, err := os.Stat("./_fixtures/coverage_fixture/coverage.txt")
+		coverFile := "./_fixtures/coverage_fixture/coverage.txt"
+		_, err := os.Stat(coverFile)
 
 		Ω(err).ShouldNot(HaveOccurred())
 
 		// Cleanup
-		os.RemoveAll("./_fixtures/coverage_fixture/coverage.txt")
+		os.RemoveAll(coverFile)
 	})
 
 	It("Works in recursive mode", func() {
-		session := startGinkgo("./_fixtures/combined_coverage_fixture", "-r", "-cover", "-coverprofile=coverage.txt")
+		session := startGinkgo("./_fixtures/combined_coverage_fixture", "-r", "-cover", "-coverprofile=coverage1.txt")
 
 		Eventually(session).Should(gexec.Exit(0))
 
 		packages := []string{"first_package", "second_package"}
 
 		for _, p := range packages {
-			coverFile := fmt.Sprintf("./_fixtures/combined_coverage_fixture/%s/coverage.txt", p)
+			coverFile := fmt.Sprintf("./_fixtures/combined_coverage_fixture/%s/coverage1.txt", p)
```

Comment:
I wonder whether we can come up with more self-explaining names for these files than 1,2,3 etc. e.g. `coverage-recursive.txt`

I'm also not super clear on why these all have different names rather than different output dirs but I'm definitely missing something.